src/tests/Microsoft.PowerFx.Core.Tests/LexerTests.cs

```diff
@@ -568,5 +568,71 @@ public void TestUnsupportedDecimalSeparatorCausesFallback()
 
             CultureInfo.CurrentCulture = oldCulture;
         }
+
+        [Fact]
+        public void TestStringInterpolation()
+        {
+            Token[] tokens;
+            string value;
+
+            value = "$\"Hello {name}\"";
+            tokens = TexlLexer.LocalizedInstance.LexSource(value);
+            Assert.NotNull(tokens);
+            Assert.Equal(7, tokens.Length);
+            Assert.Equal(TokKind.StrInterpStart, tokens[0].Kind);
+            Assert.Equal(TokKind.StrLit, tokens[1].Kind);
+            Assert.Equal(TokKind.IslandStart, tokens[2].Kind);
+            Assert.Equal(TokKind.Ident, tokens[3].Kind);
+            Assert.Equal(TokKind.IslandEnd, tokens[4].Kind);
+            Assert.Equal(TokKind.StrInterpEnd, tokens[5].Kind);
+            Assert.Equal(TokKind.Eof, tokens[6].Kind);
```

Comment:
nit: Could we use a test helper to make this more streamlined. ie:

AssertTokens("$\"Hello {name}\"", 
    TokKind.StrInterpStart, 
    TokKind.StrLit, 
 ....);  

AssertTokens(string text, params TokKind[] expected);

Reply:
Done